README.md

```
@@ -130,4 +120,32 @@ main().catch((exception) => {
 Run `node scripts/node.js` to start the app, or `webpack scripts/app.js -o assets/app.min.js` to
 build a browser version.
 
+## Resources
+| Doc | Purpose |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)